nanome/plugin_init.py

```diff
@@ -47,6 +48,9 @@ def main():
                 content = content.replace('{{%s}}' % key, value)
             with open(file_path, 'w', encoding='utf-8') as f:
                 f.write(content)
+            if file_path.endswith('.sh'):
+                perm = os.stat(file_path).st_mode
+                os.chmod(file_path, perm | 0o111)
```

Comment:
It would be clearer with flags. You're trying to do a stat.S_IEXEC here, right? Plus, that would make you use the stat that you imported but haven't used :)

Reply:
`stat.S_IEXEC` only changes it for user, to do `chmod +x`, would have to do `stat.S_IXUSR | stat.S_IXGRP | stat.S_IXOTH`. Octal notation here is cleaner, removing `stat` import.